docusaurus/docs/overview/feature-model/scoping-rules.mdx

```diff
@@ -21,27 +26,27 @@ vary according to the day of the week.
 
 The Overture Transportation theme's schema model of the transportation
 network uses two related concepts to capture the partial application of
-facts and rules: Scoped values, and Rule-based properties.
+facts and rules: scoped values, and rule-based properties.
 
-## Scoped values and Scoping properties
+## Scoped values and scoping properties
 
-A *Scoped* value is a value which only applies within a limited scope.
+A *scoped* value is a value which only applies within a limited scope.
 Most scoped values are rules in the rule lists of
-[Rule-based properties](#rules-and-rule-based-properties). However,
+[rule-based properties](#rules-and-rule-based-properties). However,
 scoped values also exist outside of rule-based properties. For
 example a signpost property belonging to a road segment might be
 geometrically scoped to its position along the road.
 
 The scope in which a scoped value applies is controlled by one or more
-special child properties of the value known as *Scoping* properties.
+special child properties of the value known as *scoping* properties.
 
 ### Geometric scoping (linear referencing)
 
 <Tabs>
   <TabItem value="description" label="Description" default>
 
 The geometric scoping property `at` limits the scope of its parent value to a
-position or range of positions along a Segment's geometry. When the parent value
+position or range of positions along a segment's geometry. When the parent value
 is a rule object, the rule only matches the position or range of positions
```

Comment:
This paragraph highlights the fact that the page content was originally written from an *exclusively* transportation focus, and probably should be massaged to be more generically-focused.

The problem with this paragraph is it assumes "your brain context = transportation" so it starts talking about segments, which are a transportation context. Maybe in this particular example, what is meant is "linear features"?

Probably the whole page needs to be audited carefully from that perspective.